app/modules/wifi.c

```diff
@@ -298,6 +298,19 @@ static int wifi_getphymode( lua_State* L )
   return 1;
 }
 
+// Lua: wifi.setmaxtxpower()
+static int wifi_setmaxtxpower( lua_State* L )
+{
+  unsigned power;
+  power = luaL_checkinteger( L, 1 );
+
+  if ( power > 82 )
```

Comment:
Please use [luaL_argcheck()](http://pgl.yoyo.org/luai/i/luaL_argcheck) for checking function parameters.

Reply:
I'll look at argcheck.  I don't see a reason to check for less than zero, though, because it's an unsigned number. 

Should I submit a pull request for other wifi functions which are missing argcheck?

Reply:
Ah, I see that the function that didn't have arg checks was explicitly checking values against an enum.